testdata/fastqs/tiny/SRR6109255_4.fastq.gz

```diff
@@ -0,0 +1 @@
+SRR6109255_3.fastq.gz
```

Comment:
I'm just confused there

Reply:
This is a symbolic link.  I wanted to preserve disk space.